pnpm-workspace.yaml

```diff
@@ -1,3 +1,2 @@
 packages:
-  - 'packages/*'
   - 'docs'
```

Comment:
For now, keeping this as a monorepo, since we will also have a `snippets/` directory, to keep things separated.

Reply:
I was thinking that we can move away from the monorepo setup for the docs. For the Node snippets, I would expect a separate package, this way we avoid polluting the root `node_modules` and we can install only the documentation dependencies. 🤔